chore: be able to hide webServer stderr (#23063)
microsoft/playwright: docs/src/test-api/class-testconfig.md

  - `timeout` ?<[int]> How long to wait for the process to start up and be available in milliseconds. Defaults to 60000.
  - `reuseExistingServer` ?<[boolean]> If true, it will re-use an existing server on the `port` or `url` when available. If no server is running on that `port` or `url`, it will run the command to start a new server. If `false`, it will throw if an existing process is listening on the `port` or `url`. This should be commonly set to `!process.env.CI` to allow the local dev server when running tests locally.
  - `stdout` ?<["pipe"|"ignore"]> If `"pipe"`, it will pipe the stdout of the command to the process stdout. If `"ignore"`, it will ignore the stdout of the command. Default to `"ignore"`.

>   - `stderr` ?<["pipe"|"ignore"]> If `"pipe"`, it will pipe the stderr of the command to the process stderr. If `"ignore"`, it will ignore the stderr of the command. Default to `"pipe"`.

Reviewer: ```suggestion
  - `stderr` ?<["pipe"|"ignore"]> Whether to pipe the stderr of the command to the process stderr or ignore it. Defaults to `"pipe"`.
```
